### PR TITLE
[v6r15] RabbitMQ module

### PR DESCRIPTION
**Pull request by Sbalbp**
Created a RabbitMQ module using stomp to both send and receive messages from the queues

### CODE REVIEW

File: Core/Utilities/RabbitMQ.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+"""
+Class for management of RabbitMQ connections
+"""
+
+__RCSID__ = "$Id$"
+
+import datetime, threading, json, stomp
```
Comment (chaen) [inline Core/Utilities/RabbitMQ.py]: you should do the import separately, one by line

File: Core/Utilities/RabbitMQ.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    return S_OK( 'Setup successful' )
+
+  def send( self, message ):
+    """
+    Sends a message to the queue
+    message contains the body of the message
+    """
+    try:
+      if type( message ) == list:
```
Comment (chaen) [inline Core/Utilities/RabbitMQ.py]: isinstance

File: Core/Utilities/RabbitMQ.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  def receive( self ):
+    """
+    Retrieves a message from the queue ( if any )
+    Returns S_ERROR if there are no messages in the queue
+    """
+    if not self.receiver:
+      return S_ERROR( 'Instance not configured to receive messages' )
+
+    with self.lock:
+      if len( self.msgList ) > 0:
```
Comment (chaen) [inline Core/Utilities/RabbitMQ.py]: if self.msgList:

**Comment by Sbalbp** [comment]
Closing this PR to open it on the actual v6r15 branch ( adding new features to the module too! )

**Pull request closed without merging**